scripts/lang/fr/__init__.py

```diff
@@ -533,7 +544,7 @@
     "smcp": "small_caps(parts[1])",
     # {{SIC}}
     # {{sic !}}
-    "SIC": "f'<sup>[sic : {parts[1]}]</sup>' if len(parts) > 1 else '<sup>[sic]</sup>'",
```

Comment:
Is it on purpose? The closing tag is still present then.

Reply:
No it's not...